apps/search-poc/src/containers/DashboardList.tsx

```diff
@@ -65,33 +63,10 @@ const DashboardListContainer: React.FunctionComponent<{
     projectLabel: string,
     viewId: string,
     dataQuery: string,
+    dashboardId: string,
   ) => void;
-  history: History;
-}> = ({ onDashboardSelected, history }) => {
-  // they might be an active dashboard id setup as a query string
-  const defaultActiveDashboardId = queryString.parse(history.location.search)
-    .dashboard;
-
-  // default active dashboard is the first one if none is present in querystring
-  const [activeDashboard, setActiveDashboard] = React.useState<{
-    dashboard: DashboardContainer;
-    view: View;
-  }>(
-    defaultActiveDashboardId
-      ? getDashBoardConfig(defaultActiveDashboardId.toString(), dashboardConfig)
-      : dashboardConfig[0],
-  );
-
-  // call callback prop when dashboard is selected
-  React.useEffect(() => {
-    onDashboardSelected(
-      activeDashboard.view.orgLabel,
-      activeDashboard.view.projectLabel,
-      activeDashboard.view.viewId,
-      activeDashboard.dashboard.dataQuery,
-    );
-  }, [activeDashboard, onDashboardSelected]);
```

Comment:
just remove `onDashboardSelected` in order to fix that loop issue (but it will cause a lint error)